### PR TITLE
docs: add logical volume to storage options

### PR DESCRIPTION
**Pull request by satoru-takeuchi**
**Description of your changes:**

Rook supports osd on logical volume in host-based clusters. Let's add it to the list of valid device types.

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/Contributing/development-flow/#commit-structure)).
- [x] **Skip Tests for Docs**: If this is only a documentation change, add the label `skip-ci` on the PR.
- [x] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/Contributing/development-flow/#submitting-a-pull-request)
- [x] [Pending release notes](https://github.com/rook/rook/blob/master/PendingReleaseNotes.md) updated with breaking and/or notable changes for the next minor release.
- [x] Documentation has been updated, if necessary.
- [x] Unit tests have been added, if necessary.
- [x] Integration tests have been added, if necessary.

### CODE REVIEW

File: Documentation/Getting-Started/Prerequisites/prerequisites.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ In order to configure the Ceph storage cluster, at least one of these local stor
 
 * Raw devices (no partitions or formatted filesystems)
 * Raw partitions (no formatted filesystem)
+* Logical Volumes (no formatted filesystem)
```
Comment (galexrt) [inline Documentation/Getting-Started/Prerequisites/prerequisites.md:22]: If we are talking about logical volume as in "LVM" let's add it in brackets to it that it is clear what is meant here.

Comment (satoru-takeuchi) [reply Documentation/Getting-Started/Prerequisites/prerequisites.md:22]: How about `LVM Logical Volume (no formatted filesystem)`? I've seen this words many times.

Comment (galexrt) [reply Documentation/Getting-Started/Prerequisites/prerequisites.md:22]: Sounds good!

**Comment by galexrt** [comment]
Added skip ci label as this a docs only change

**Comment by satoru-takeuchi** [comment]
> Added skip ci label as this a docs only change

Thanks, and fixed.

**Review by galexrt — APPROVED** [review]

**Pull request merged**